src/xmipp/libraries/reconstruction/volume_align_prog.cpp

```diff
@@ -400,7 +404,21 @@ class ProgAlignVolumes : public XmippProgram
     		PyObject * pFunc = getPointerToPythonFRMFunction();
     		double rot,tilt,psi,x,y,z,score;
     		Matrix2D<double> A;
-    		alignVolumesFRM(pFunc, params.V1(), params.V2(), Py_None, rot,tilt,psi,x,y,z,score,A,maxShift,maxFreq,params.mask_ptr);
+    		if(starting_tilt!=-90 || ending_tilt!=90){
+    			std::cout<<"you are compensating for the missing wedge, the first volume should be rotated with 90 degrees about the y-axis"<<std::endl;
+    			PyObject * pSTMMclass = getPointerToPythonSingleTiltWedgeClass();
+    			PyObject * arglist = Py_BuildValue("(ii)", starting_tilt , ending_tilt);
+    			PyObject * SingleTiltWedgeMask = PyObject_CallObject(pSTMMclass, arglist);
+    			// The order of volumes has to be flipped in order to compensate for a single tilt missing wedge. For those who are not using this mask, no changes in results will happen.
+    			alignVolumesFRM(pFunc, params.V2(), params.V1(), SingleTiltWedgeMask, rot,tilt,psi,x,y,z,score,A,maxShift,maxFreq,params.mask_ptr);
+    			std::cout<<"Reverse your usage of the option --inverse when using xmipp_transform_geometry"<<std::endl;
```

Comment:
? What do you mean by this?

Reply:
This issue is rather a bit deep.
alignVolumesFRM is a wrapper for a python function (pFunc is a pointer to it).
alignVolumesFRM has two issues that I am rotating about as follows (I can't touch it because it is used in other places as [here](https://github.com/I2PC/xmipp/blob/devel/src/xmipp/libraries/parallel/mpi_classify_CLTomo_prog.cpp)):
The program (the python version) takes two volumes as input (3D numpy matrices), and when they are passed from the wrapper, they get rotated (the origin reference is not the same). This issue becomes a problem ONLY when we try to compensate for the missing wedge (direction here matters, normally doesn't matter). And I am solving it by rotating the volume with 90 degrees and displaying a message.
The other issue, is that the original function takes 3 masks (see frm_align [here](https://github.com/I2PC/xmipp/blob/devel/src/xmipp/external/sh_alignment/python/frm.py)) but the wrapper is made to pass 2 masks. I had to compensate for this issue by passing these volumes in reverse order (keeping the default usage untouched). However, when things are aligned in the reverse order, we should use the results of the alignment using "xmipp_transform_geometry" with the option --inverse 'used' (if normally not used) or unused (if normally used).

Reply:
So you wanted to say sth like:
`std::cout<<"Do not use '--inverse' flag while applying transforms using xmipp_transform_geometry"<<std::endl;` ?
Btw be careful. While alignment _a_ of _A to B_ should  be equivalent to _a<sup>-1</sup>_ of _B to A_, in praxis this might not be true (depending on your algorithm)

Reply:
Not exactly, I am trying to tell users to invert their usage of the option '--inverse', i.e, to use it if they usually don't, or to not use it if they usually do.
Thanks for the warning, I tested this program (with the algorithm of frm where I am adding the option), and it does as expected (forward and inverse alignment are both valid).

Reply:
Then I suggest sth like:
`std::cout<<"If you intend to apply transform using xmipp_transform_geometry, use --inverse flag (if it was not present before), or remove it (if it was present before)"<<std::endl;`

Reply:
Thanks, this looks very good.